src/components/Quote.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+const Quote = ({ apiKey }) => {
+  const [quote, setQuote] = useState(undefined);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    const fetchQuotes = async () => {
+      try {
+        const response = await fetch(
+          'https://api.api-ninjas.com/v1/quotes?category=happiness',
+          {
+            headers: { 'X-Api-Key': apiKey },
+            method: 'GET',
+            contentType: 'application/json',
+          },
+        );
+
+        const data = (await response.json())[0];
+
+        setQuote(data);
+      } catch (err) {
+        setError(true);
+      }
+      setLoading(false);
+    };
+
+    fetchQuotes();
+  });
```

Comment:
- **[OPTIONAL]** On fetch quotes it starts to load every quote too fast that I cannot read them, please 🙏🏽 correct this behavior for a better UX.